src/transform/tfidf/script.py

```diff
@@ -0,0 +1,65 @@
+import sys
+import mudata
+import muon
+
+## VIASH START
+par = {
+    "input": "work/d9/3adbd080e0de618d44b59b1ec81685/run.output.h5mu",
+    "output": "output.h5mu",
+    "scale_factor": 10000,
+    "modality": "atac",
+    "input_layer": None,
+    "output_layer": None,
+    "output_compression": "gzip",
+    "log_idf": True,
+    "log_tf": True,
+    "log_tfidf": False
+}
+meta = {"functionality_name": "lognorm"}
+## VIASH END
+
+sys.path.append(meta["resources_dir"])
+# START TEMPORARY WORKAROUND setup_logger
+# reason: resources aren't available when using Nextflow fusion
+# from setup_logger import setup_logger
+def setup_logger():
+    import logging
+    from sys import stdout
+
+    logger = logging.getLogger()
+    logger.setLevel(logging.INFO)
+    console_handler = logging.StreamHandler(stdout)
+    logFormatter = logging.Formatter("%(asctime)s %(levelname)-8s %(message)s")
+    console_handler.setFormatter(logFormatter)
+    logger.addHandler(console_handler)
+
+    return logger
+# END TEMPORARY WORKAROUND setup_logger
+logger = setup_logger()
+
+logger.info("Reading input mudata")
+mdata = mudata.read_h5mu(par["input"])
+mdata.var_names_make_unique()
```

Comment:
I prefer the component not altering the var names. I would accept raising an error if this is a requirement of the input. In the pipeline there could be a component that does this once at the beginning of the pipeline. But I would like to avoid doing this everywhere. 
```suggestion
```